radix-engine/src/transaction/validator.rs

```diff
@@ -8,15 +8,23 @@ use crate::errors::*;
 use crate::model::*;
 
 pub fn validate_transaction(
-    transaction: &Transaction,
+    transaction: &SignedTransaction,
```

Comment:
Minor: Seems like this can just be moved into SignedTransaction?

Reply:
Yeah, let me move it there